lib/language_pack/rails2.rb

```diff
@@ -45,7 +45,8 @@ def default_process_types
 
   def compile
     instrument "rails2.compile" do
-      install_plugins
+      # XXX(dmauldin): we do not wish for you to stomp our logger object, thanks
+      # install_plugins
```

Comment:
The only other plugin is this one: https://github.com/heroku/rails_serve_static_assets

It's apparently required for Rails 4, but we're using Rails 3, which has a config flag for serving static assets.  Anything that was pre-compiled will be served up.  I'm not sure what changed about that in Rails 4, but again, we're not using it.

We did not use either of these plugins when using stackato v2.  Things should continue to work fine without using the plugins.